sunsynk/definitions.py

```diff
@@ -179,6 +227,12 @@
 PROG5_TIME.max = PROG6_TIME
 PROG6_TIME.max = PROG1_TIME
 
+PROG_CHARGE_OPTIONS = {
+    4: "No Grid or Gen",
+    5: "Allow Grid",
+    6: "Allow Gen",
+    7: "Allow Grid & Gen",
+}
```

Comment:
The Sunsynk and Deye inverter UI have these as two checkboxes, but they operate on one register. I could try to have two switches operate on one register (Prog N Grid Charge, Prog N Gen Charge) but I haven't seen an example on how two different sensors can operate on the same register on the codebase.

Reply:
There are no other sensors that only use certain bits of a register today. Let's leave it like this for now